lib/mix/lib/releases/assembler.ex

```diff
@@ -497,6 +505,12 @@ defmodule Mix.Releases.Assembler do
       do: :ok
   end
 
+  defp generate_cookie do
+    # 255 is max. atom length, see http://erlang.org/doc/efficiency_guide/advanced.html
+    # Cookies, at least when read from disk, need to be from the printable ASCII range
+    (1..255)|> Enum.map(fn _ -> Enum.random(?\s..?~) end) |> List.to_atom
```

Comment:
RE: spaces, yeah let's omit them from generated cookies, but it's good to know that they are supported in any case.

I'm open to using `crypto:strong_rand_bytes/1`, but I have two things that make me hesitate:

1. It depends on OpenSSL being available, which we can maybe assume will be available, but I'm not sure to what degree that complicates Windows support, if at all. I still want to keep Distillery merge-able into Mix as well, at least in the near term, and I suspect such a dependency would not be desired.
2. The binary generated can contain unprintable characters, which we could probably solve by sha256 hashing the binary, so maybe it's not an issue

I also want to be clear that while I want to generate reasonably secure cookies, nodes should not be remotely accessible via the internet, and I have no plans to protect that use case out of the box - if someone wants to do that, that is their prerogative, but I can't ever guarantee the defaults will be perfectly secure. This cookie should be sufficiently random to prevent collisions (within reason), printable, and keep distillery free of any additional dependencies, and that's about as far as I wish to take any defaults provided.